### DISCUSSION TITLE
How to revert hnome to defaults?

### DISCUSSION THREAD

**Discussion by Dani3I** [SpiralLinux/SpiralLinux-project#401] [Q&A] [answered]
I would like default gnome experience. How to delete all modifications made by spiral linux so everything is klike on, for example, fedora (icons, no bar on the left etc)?

**Comment by geckolinux** [comment, marked as answer]
Hi there, you can delete `~/.config/dconf` and `~/.config/gtk-3.0` and `~/.config/gtk-4.0` .
